### ISSUE TITLE
UI: timeout of 60000ms exceeded

### ISSUE THREAD

**Issue by albertofilice** [stackrox/stackrox#3905]
in browsing by policy "vulnerability-management/policies" if you try to view the reports of a policy with many violations, for example a policy with 1796 failing deployments, you run into the timeout error

<img width="743" alt="image" src="https://user-images.githubusercontent.com/35273403/203818613-328f5d0c-ba58-4016-8612-2810e9a79118.png">

I think it could be fixed easily by changing the value in this file:

ui/apps/platform/src/configureApolloClient.js

```
        // set a long timeout for GraphQL requests, as an escape hatch for slow queries
        const modifiedConfig = { ...config, timeout: 60000 };
```

<img width="1438" alt="image" src="https://user-images.githubusercontent.com/35273403/203820066-f7c0e8e7-f2b3-45fc-8294-75674b05adc7.png">

I have noticed that the error comes especially with these two api calls:

/api/graphql?opname=get​​PolicyDEPLOYMENT

/api/graphql?opname=getPolicy

**Comment by stehessel** [comment]
Hey @albertofilice, thanks for the report and the potential fix. I let our UI experts know to take a look at your suggestion.

**Comment by dvail** [comment]
Hello @albertofilice, we've identified a couple of problems with pagination in this section that are leading to the long request time and are looking for potential fixes. The increase in timeout value may work in a pinch, but the issue could reappear as the number of deployments grows.

**Comment by albertofilice** [comment]
Hello @dvail, thanks for reporting, do you know in which version these bugs will be fixed?

**Comment by dvail** [comment]
We're testing some fixes now that should hopefully be merged in the next couple of days and ready for the 3.74 release.

**Comment by albertofilice** [comment]
Thanks a lot for the information and the update.

**Comment by dvail** [comment]
The two issues regarding pagination have been resolved, tested against a simulated scenario with 30,000+ deployments, and merged. The fixes will be available in the next release (3.74).
